src/detectors/BHCAL/BHCAL.cc

```diff
@@ -42,13 +49,13 @@ extern "C" {
         app->Add(new JChainMultifactoryGeneratorT<CalorimeterHitReco_factoryT>(
           "HcalBarrelRecHits", {"HcalBarrelRawHits"}, {"HcalBarrelRecHits"},
           {
-            .capADC = 65536,
-            .dyRangeADC = 1.0 * dd4hep::GeV,
-            .pedMeanADC = 10,
-            .pedSigmaADC = 2.0,
-            .resolutionTDC = 1.0 * dd4hep::picosecond,
-            .thresholdFactor = 5.0,
-            .thresholdValue = 1.0,
+            .capADC        = HcalBarrel_capADC,
+            .dyRangeADC    = HcalBarrel_dyRangeADC,
+            .pedMeanADC    = HcalBarrel_pedMeanADC,
+            .pedSigmaADC   = HcalBarrel_pedSigmaADC, // not used; relying on energy cut
```

Comment:
Apologies, but I wasn't able to make the background meeting yesterday. Was there a decision to use the energy threshold vs. the ADC threshold?

Since we'll cut on ADC in actual data, I think it would be better to move to relying on only the ADC threshold.

Reply:
There hasn't been a decision, just to go through the calo ones individually and where necessary/easy default to the energy cut. If you prefer the ADC cut, just give me the okay (and add the number to the spreadsheet), and I will make the change.

Reply:
Ah! Gotcha! In that case, let's plan on going to ADC only. Let me double check that the 2 are indeed equivalent, and then we can make the changes accordingly...

Reply:
Hi Kolja, I finished up running some checks. I've attached a plot comparing the reconstructed hit spectrum for different combinations of energy cuts/adc thresholds (made w/ 100 10 GeV single pi-).

I should have caught this earlier, but we do want to cut on a half-MIP (5 MeV) deposition without any adjustment for sampling fractions (like John noted). So I think I may be misunderstanding what these parameters are...

We've set the pedMeanADC to be 300 ADC, just under 5 MeV. Following the logic in the digi factory, the condition for zero-suppression is:

`amplitude < (threshold + pedMeanADC)`

which means that if we want to cut anything below 300 (333) ADC then we would need to set the threshold parameters to something like

```
thresholdFactor = 0 (1)
thresholdValue = 0 (31)
```

In the plot, the yellow line corresponds to to a zero-suppression of 300 (factor/value = 0) while the green line corresponds to a zero-suppression of 333 (factor = 1, value = 31).

So I was wondering: do `thresholdFactor` and `thresholdValue` correspond to physical quantities? As I understand it, `thresholdFactor` corresponds to an n-sigma cut on our pedestal distribution, but I'm confused about what `thresholdValue` corresponds to...

![checkingCuts_withVal31fac1AndVal0Fac0_PR1095 bhcalRecHitsEne d3m11y2023](https://github.com/eic/EICrecon/assets/35080855/42199422-a401-4e8c-9902-407711b81fa9)